tests/genai/judges/optimizers/conftest.py

```diff
@@ -47,117 +47,181 @@ def get_input_fields(self) -> list[JudgeField]:
         """Get the input fields for this mock judge."""
         return [
             JudgeField(name="inputs", description="Test inputs"),
+            JudgeField(name="outputs", description="Test outputs"),
         ]
 
 
-@pytest.fixture
-def mock_judge():
-    """Create a mock judge for testing."""
-    return MockJudge(model="openai:/gpt-3.5-turbo")
-
+def _create_trace_helper(
```

Comment:
`test_trace_to_dspy_example_success` and `test_trace_to_dspy_example_missing_required_fields` in `tests/genai/judges/optimizers/test_dspy_utils.py` require traces with inputs, others with outputs (no inputs), with / without expectations, etc.

I've defined a reusable helper utility that reduces the # of lines of code required to define these traces